diktat-gradle-plugin/build.gradle.kts

```diff
@@ -6,6 +6,7 @@ plugins {
     kotlin("jvm") version "1.4.31"
     jacoco
     id("pl.droidsonroids.jacoco.testkit") version "1.0.7"
+    id("org.gradle.test-retry") version "1.2.0"
```

Comment:
The latest version is [1.2.1](https://plugins.gradle.org/plugin/org.gradle.test-retry). Why do we use 1.2.0?

Reply:
Lol, it was released yesterday and I haven't seen it yet :) Nice catch, will change